src/Trakt/Provider.php

```diff
@@ -75,6 +75,6 @@ protected function getTokenFields($code)
      */
     public static function additionalConfigKeys()
     {
-        return ['api_version'];
+        return ['api_version', 'client_id'];
```

Comment:
Think this is also covered by the default config items

Reply:
See 565ae93ad50d6f97970000878c940f3e0eb4b707